app-marketplace-v2-spec/architecture.md

```diff
@@ -48,8 +53,11 @@
 3. There are 3 kinds of commands:
 - [Slash commands](commands.md#slash-commands) are entered from the message box, with autocomplete.
 - [Embedded commands](commands.md#embedded-commands) are what was previously known as Post Actions or Interactive Messages. They can be embedded in Post Slack attachments, and #STRETCH markdown. **Buttons** launch fully-configured commands, **checkboxes** toggle boolean flags, and **selects** allow to choose a value from a list.
+  - Dani Q: This will include other UI elements, like what I have been working for mobile, right? (channel header button, channel header menu, main menu ...)
```

Comment:
I am all in for having a stronger spec of the locations before working on them. So far I have been trying to keep consistent with the plugin API implementations, and also the buttons on the nearby in the UI. But since this is something completely new, I think we should break with the old and just look at what we need.